cypress/e2e/move.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Test component movement
+  Background:
+    Given I clear localstorage
+    And I visit the "/"
+
+  Scenario: Components should be draggable
+    When I drag "svg#root svg#external1" to 500,500
```

Comment:
Can you check the position of the element before to make sure you move it ?
Like:
`Given I expect "svg#root svg#external1" to be at position 0,0`

Reply:
done